<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowJacobiDemo</class>
 <widget class="QMainWindow" name="MainWindowJacobiDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>996</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jacobi Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Display options</string>
       </property>
       <widget class="QCheckBox" name="checkBoxColModel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Collision model</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButtonLoad">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>(Re-)Load</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonBox2TCP">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset Boxes</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonJacobiTest">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Jacobi TCP</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>181</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Kinematic Chain</string>
       </property>
       <widget class="QLabel" name="label_TCP">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TCP:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_NrJoints">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nr of joints:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxKC">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButtonJacobiTest_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>320</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Jacobi 2 TCP</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>390</y>
         <width>171</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>TCP 1</string>
       </property>
       <widget class="QSlider" name="horizontalSliderA">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderX">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>b</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderB">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>g</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderG">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderY">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderZ">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>540</y>
         <width>171</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>TCP 2</string>
       </property>
       <widget class="QSlider" name="horizontalSliderA_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderX_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>b</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderB_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>g</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderG_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderY_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderZ_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>730</y>
         <width>171</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>Elbow</string>
       </property>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>a</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderX_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderY_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderZ_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderA_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>b</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderG_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>g</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderB_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButtonJacobiTest_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Jacobi TCP Elbow(Z)</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
